src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$side: 100px;
+$number-of-date: 7;
+$paddings: 10px;
+$gap: 1px;
+$days:
+  "Monday" 1,
```

Comment:
Please use the short versions of the day names (mon, tue, wed, thu, fri, sat, sun) as specified in the task description.